.github/workflows/test.yaml

```
@@ -30,23 +30,19 @@ jobs:
       - name: change mysql charset
         run: mysql -h 127.0.0.1 -P 3307 -u root -p'root-password' -e "ALTER DATABASE testdb CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci"
 
-      - uses: actions/setup-python@v4
+      - name: Setup uv
```

actionlint (1.7.9)

step must run script with "run" section or run action with "uses" section

(syntax-check)